services/wallet/transfer/commands.go

```diff
@@ -166,7 +167,7 @@ func (c *erc20HistoricalCommand) Run(ctx context.Context) (err error) {
 	for !c.iterator.Finished() {
 		headers, _, _, err := c.iterator.Next(ctx)
 		if err != nil {
-			log.Error("failed to get next batch", "error", err)
+			log.Error("failed to get next batch", "error", err, "chainID", c.chainClient.NetworkID()) // TODO: stop inifinite command in case of an error that we can't fix like missing trie node
```

Comment:
How about connecting this TODO with a GitHub issue to increase awareness? Similar to [this](https://github.com/status-im/status-desktop/blob/ccd8c5b65f973718aa935d37736edc90b763957f/ui/app/AppLayouts/Wallet/views/walletconnect/WalletConnectSDK.qml#L3). Otherwise all these todos might be fade out in the big codebase.